tests/e2e/cypress/integration/Autologin/01-autologin/index.ts

```diff
@@ -0,0 +1,171 @@
+import { When, Then, Given } from 'cypress-cucumber-preprocessor/steps';
+
+import { removeContact, initializeConfigACLAndGetLoginPage } from '../common';
+
+let link = '';
```

Comment:
Use Aliases instead of `let`: https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Sharing-Context